Add Skoob icon

juandesenvolvedor opened the pull request:
i Add Skoob icon to the repository.

<!--
Before opening your pull request, have a quick look at our contribution guidelines:
https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md

Consider adding a preview image of your submission using:
https://wasm.simpleicons.org/preview
-->

![skoobPR](https://github.com/simple-icons/simple-icons/assets/95597422/55cb25f1-96a1-4b24-99bd-05553e975ab4)


**Issue:** closes #10079 

**Similarweb rank:** 24,211

<!--
The Similarweb rank can be retrieved at https://www.similarweb.com
Please see our contributing guidelines for more details on how we assess a brand's popularity.
-->

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

<!--
Anything relevant, for example:
  - Why did you pick the hex value?
  - Did you manually vectorize the logo?
  - Have you used multiple sources?
  - etc.
-->

PeterShaggyNoble commented:
@juandesenvolvedor, no need to keep closing your PRs; you can just continue your work in the one. If you come across problems or have questions then you can ask for help in the comments.

juandesenvolvedor commented:
sorry, i'm new at contributing, i'm facing some problems, i was doing everything acording to the contributing guide, did some adjustments on the svg to fix it, fixed the svg code and added the title, hex and source to the documentation, but when i try to pull three checks show error, do you know what can i do to fix it?

PeterShaggyNoble commented:
@juandesenvolvedor, that error was merely due to having a trailing newline in the SVG file. If you're working through the GitHub website then that's unavoidable; even if you explicitly remove it GitHub will still automatically add it back in. If you'd like to reopen/resubmit this PR then you can ignore that error and we'll remove the newline for you prior to merging.
